Automattic/jetpack
# AI Assistant: Group options under `Write with AI`

Closes #31039 

## Proposed changes:

* Move options under `Write with AI`.
* Improve UX and texts into `Write with AI`.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Insert `AI Assistant`.
* You should see only `Write with AI` option.
* It should not appear if has content in the block and the options should be context based.

### Demo


https://github.com/Automattic/jetpack/assets/1663717/bee05af0-7fcb-4f19-ab83-cad30759f95a
